arkouda/pdarrayclass.py

```diff
@@ -4023,7 +4045,8 @@ def mod(dividend, divisor) -> pdarray:
     return dividend % divisor
 
 
-@typechecked
+# at typechecked -- typechecked causes erroneous error messages, and we do our own below
```

Comment:
If you want to leave `@typechecked` off this PR, can you make a ticket to add it back in?

Reply:
I must take better notes.  I don't recall what errors typechecking was causing, so I put it back in.  Seems fine.

Reply:
Thanks!